components/dropdown/test/dropdown-menu.visual-diff.js

```diff
@@ -44,25 +50,28 @@ describe('d2l-dropdown-menu', () => {
 	});
 
 	it('with-header-footer', async function() {
+		this.test.value = '#dropdown-menu-header-footer'; // Needed for retries
 		await open(page, '#dropdown-menu-header-footer');
 		const rect = await getRect(page, '#dropdown-menu-header-footer');
 		await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
 	});
 
 	it('with-header-footer-mobile', async function() {
 		await page.setViewport({ width: 300, height: 800 });
+		this.test.value = '#dropdown-menu-header-footer-mobile'; // Needed for retries
 		await open(page, '#dropdown-menu-header-footer-mobile');
 		await visualDiff.screenshotAndCompare(page, this.test.fullTitle());
-		await reset(page, '#dropdown-menu-header-footer-mobile');
```

Comment:
Anything that happens after `screenshotAndCompare` won't run if the test fails, but `afterEach` will